deploy/07_erc721drop.ts

```diff
@@ -30,55 +30,155 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
+    let editionsMetadataRendererProxy = await genesisDeployHelper(
       hre,
       salt,
-      'HolographFeeManager',
-      generateInitCode(['uint256', 'address'], [feeBPS, deployer.address]), // initCode
-      futureFeeManagerAddress
+      'EditionsMetadataRendererProxy',
+      generateInitCode(['address', 'bytes'], [futureEditionsMetadataRendererAddress, generateInitCode([], [])]),
+      futureEditionsMetadataRendererProxyAddress
     );
   } else {
-    hre.deployments.log('"EditionMetadataRenderer" is already deployed.');
+    hre.deployments.log('"EditionsMetadataRendererProxy" is already deployed.');
   }
 
-  // Metadata renderer
-  const futureEditionMetadataRendererAddress = await genesisDeriveFutureAddress(
+  // Deploy the HolographDropsEditionsV1 custom contract source
+  const holographDropsEditionsV1InitCode = generateInitCode(
+    [
+      'tuple(address,address,address,address,uint64,uint16,tuple(uint104,uint32,uint64,uint64,uint64,uint64,bytes32),address,bytes)',
+    ],
+    [
+      [
+        '0x0000000000000000000000000000000000000000', // holographERC721TransferHelper
+        '0x0000000000000000000000000000000000000000', // marketFilterAddress (opensea)
+        deployer.address, // initialOwner
+        deployer.address, // fundsRecipient
+        0, // 1000 editions
+        1000, // 10% royalty
+        [0, 0, 0, 0, 0, 0, '0x' + '00'.repeat(32)], // setupCalls
+        futureEditionsMetadataRendererProxyAddress, // metadataRenderer
+        generateInitCode(['string', 'string', 'string'], ['decscription', 'imageURI', 'animationURI']), // metadataRendererInit
+      ],
+    ]
+  );
+  const futureHolographDropsEditionsV1Address = await genesisDeriveFutureAddress(
     hre,
     salt,
-    'EditionMetadataRenderer',
-    generateInitCode([], []) // initCode
+    'HolographDropsEditionsV1',
+    holographDropsEditionsV1InitCode
   );
-  hre.deployments.log('the future "EditionMetadataRenderer" address is', futureEditionMetadataRendererAddress);
-  let editionMetadataRendererDeployedCode: string = await hre.provider.send('eth_getCode', [
-    futureEditionMetadataRendererAddress,
+  hre.deployments.log('the future "HolographDropsEditionsV1" address is', futureHolographDropsEditionsV1Address);
+
+  let holographDropsEditionsV1DeployedCode: string = await hre.provider.send('eth_getCode', [
+    futureHolographDropsEditionsV1Address,
     'latest',
   ]);
 
-  if (editionMetadataRendererDeployedCode == '0x' || editionMetadataRendererDeployedCode == '') {
-    await genesisDeployHelper(
+  if (holographDropsEditionsV1DeployedCode == '0x' || holographDropsEditionsV1DeployedCode == '') {
+    hre.deployments.log('"HolographDropsEditionsV1" bytecode not found, need to deploy"');
+    let holographDropsEditionsV1 = await genesisDeployHelper(
       hre,
       salt,
-      'EditionMetadataRenderer',
-      generateInitCode([], []), // initCode
-      futureEditionMetadataRendererAddress
+      'HolographDropsEditionsV1',
```

Comment:
This isn't related to this particular line, but this contract name is a bit verbose. I think we can update it to just be `HolographDropsV1`. Editions are essentially just a type of Drop so it's already implied

Reply:
Sure, we can go through the code and rename this contract.